src/main/java/uk/gov/hmcts/reform/jobscheduler/services/jobs/exceptions/JobException.java

```diff
@@ -1,8 +1,11 @@
 package uk.gov.hmcts.reform.jobscheduler.services.jobs.exceptions;
 
-public class JobException extends RuntimeException {
+import uk.gov.hmcts.reform.logging.exception.AlertLevel;
+import uk.gov.hmcts.reform.logging.exception.UnknownErrorCodeException;
+
+public class JobException extends UnknownErrorCodeException {
```

Comment:
Do we need a test to confirm all these changes?

Reply:
wouldn't think so. logging is extensively tested in delegated repo